lib/noticed.rb

```diff
@@ -30,8 +30,7 @@ def self.notify(recipients:, notification:)
       notification.notify(recipient)
     end
 
-    # Clear the recipient after sending to the group
-    notification.recipient = nil
+    clear_recipient(notification)
```

Comment:
What about adding the method to the Notification instead? 

```suggestion
notification.clear_recipient
```

Reply:
Sounds great. Let's do a public instance method on the Base class?

Reply:
I think that makes sense. 👍

Reply:
I think it's good to go. 👍